R/epichains.R

```diff
@@ -271,20 +468,13 @@ tail.epichains <- function(x, ...) {
 #' # Aggregate cases per generation
 #' cases_per_gen <- aggregate(chains, grouping_var = "generation")
 #' head(cases_per_gen)
-aggregate.epichains <- function(x,
-                                grouping_var = c(
-                                  "time",
-                                  "generation"
-                                ),
-                                ...) {
-  validate_epichains(x)
-  # Check that the object is of type "chains_tree"
-  if (!is_chains_tree(x)) {
-    stop(
-      "object must be an epichains object with 'chains_tree' attribute, ",
-      "which can be generated using the `simulate_tree()` function."
-    )
-  }
+aggregate.epichains_tree <- function(x,
+                                     grouping_var = c(
```

Comment:
would this be better as `by` to be in line with, e.g., `aggregate.data.frame`

Reply:
Fixed in f82f7235bae5df46bcec90030b1f30b51a7215f3.